mmpose/datasets/pipelines/pose3d_transform.py

```diff
@@ -206,23 +312,50 @@ class RelativeJointRandomFlip:
     """Data augmentation with random horizontal joint flip around a root joint.
 
     Args:
-        item (str): The name of the pose to flip.
-        root_index (int): Root joint index in the pose.
-        visible_item (str): The name of the visibility item which will be
-            flipped accordingly along with the pose.
+        item (str|list[str]): The name of the pose to flip.
+        flip_cfg (dict|list[dict]): Configurations of the fliplr_regression
+            function. It should specify `center_mode` and `center_x` or
+            `center_index`.
+        visible_item (str|list[str]): The name of the visibility item which
+            will be flipped accordingly along with the pose.
         flip_prob (float): Probability of flip.
+        flip_camera (bool): Whether to flip horizontal distortion coefficients.
+        camera_param (dict|None): The camera parameter dict. See the camera
+            class definition for more details. If None is given, the camera
+            parameter will be obtained during processing of each data sample
+            with the key "camera_param".
 
     Required keys:
         item
     Modified keys:
-        item
+        item (, camera_param)
     """
 
-    def __init__(self, item, root_index, visible_item=None, flip_prob=0.5):
+    def __init__(self,
+                 item,
+                 flip_cfg,
+                 visible_item=None,
+                 flip_prob=0.5,
+                 flip_camera=False,
+                 camera_param=None):
         self.item = item
-        self.root_index = root_index
+        self.flip_cfg = flip_cfg
         self.vis_item = visible_item
         self.flip_prob = flip_prob
+        self.flip_camera = flip_camera
+        if camera_param is None:
+            self.static_camera = False
+        else:
+            self.static_camera = True
+            self.camera_param = camera_param
+
+        if isinstance(self.item, str):
+            self.item = [self.item]
+        if isinstance(self.flip_cfg, dict):
+            self.flip_cfg = [self.flip_cfg]
```

Comment:
Maybe duplicate the flip_cfg for each item like: self.flip_cfg = [self.flip_cfg] * len(self.item)